apps/team-awesome-web/stories/components/Button.stories.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+
+import { Button } from "../../src/components/button";
+import { ReactComponent as PlusIcon } from "../../public/icons/plus-icon.svg";
+
+export const Primary = ({ ...args }) => (
+  <Button
+    disabled={args.disabled}
+    onClick={args.onClick}
+    variant={!args.disabled ? "primary" : args.disabled}
+  >
+    {args.icon === "Plus" ? <PlusIcon /> : null}
+    {!args.label ? "Primary button" : args.label}
+  </Button>
+);
+
+export const Secondary = ({ ...args }) => (
```

Comment:
Whey spread this?

Reply:
So apparently this is the preferred way of destructuring because I kept getting lint errors about this.